### DISCUSSION TITLE
Editor(s) save content

### DISCUSSION THREAD

**Discussion by sluc69** [joomla/joomla-cms#33684] [2. Cold Features]
Current, when saving the content of any (build-in) editor in the back-end, The page is reloaded and the editor window is reset to the top of the content. If you were somewhere in the middle of the document, you need to FIND it again scrolling through the document.
Is there a need to reload the page? 
Cannot just save the content and indicate (save button green means saved, red means changes detected) it is saved without refreshing the the whole page? It would make correcting errors and testing a lot simpler.
FF explorer in V.4 can!

I didn't find any keyboard shortcut for saving, are there any short/hot-keys?

**Comment by dgrammatiko** [comment]
> Is there a need to reload the page?

Yes Joomla is not using AJAX for saving the forms

> I didn't find any keyboard shortcut for saving, are there any short/hot-keys?

There was some code for key bindings not sure if it was reverted

**Reply by brianteeman** [reply]
It was not accepted

**Comment by sluc69** [comment]
Asking the same question again:
No AJAX for saving articles??
Still not acceptable? because you don't need it??
Make it an option and let the USER decide to use it or not!
